pubsublite/manager.go

```diff
@@ -63,12 +74,33 @@ func NewManager(cfg ManagerConfig) (*Manager, error) {
 	if err != nil {
 		return nil, fmt.Errorf("pubsublite: failed creating admin client: %w", err)
 	}
-	return &Manager{cfg: cfg, client: client}, nil
+	m := &Manager{cfg: cfg, client: client}
+
+	m.monitoringClient, err = monitoring.NewMetricClient(context.Background())
+	if err != nil {
+		return nil, fmt.Errorf("pubsublite: failed creating monitoring client: %w", err)
+	}
+
+	mp := cfg.meterProvider()
+	meter := mp.Meter("github.com/elastic/apm-queue/pubsublite")
+	consumerGroupLagMetric, err := meter.Int64ObservableGauge("pubsublite.backlog_message_count")
```

Comment:
[for reviewers] I know that @axw is keen on having the same key for lag across kafka and pubsublite instead of having "kafka.consumer_group_lag" and "pubsublite.backlog_message_count". The labels will be different though, pubsublite has subscription_id while kafka has group + topic. Will that be a problem?

Reply:
Yes that would defeat the purpose. We encode Pub/Sub Lite subscription names with abstract notion of "consumer group": https://github.com/elastic/apm-queue/blob/fa8e0138acdfb9937e61e956ed83eb1da286d14e/pubsublite/common.go#L135-L139

We should reverse that here, and treat the bit after the "+" as the consumer group name.

Reply:
Unified the metric name. What about the meter name?
```
meter := mp.Meter("github.com/elastic/apm-queue/pubsublite")
```

Reply:
We don't do anything with the meter name, so that's fine as it is for now.